packages/features/calendars/weeklyview/components/DateValues/index.tsx

```diff
@@ -5,19 +5,25 @@ import { useLocale } from "@calcom/lib/hooks/useLocale";
 import classNames from "@calcom/ui/classNames";
 
 type Props = {
+  showBorder: boolean;
+  borderColor: "default" | "subtle";
   days: dayjs.Dayjs[];
   containerNavRef: React.RefObject<HTMLDivElement>;
 };
 
-export function DateValues({ days, containerNavRef }: Props) {
+export function DateValues({ showBorder, borderColor, days, containerNavRef }: Props) {
   const { i18n } = useLocale();
   const formatDate = (date: dayjs.Dayjs): string => {
     return new Intl.DateTimeFormat(i18n.language, { weekday: "short" }).format(date.toDate());
   };
   return (
     <div
       ref={containerNavRef}
-      className="bg-default dark:bg-muted border-b-subtle rtl:border-r-default sticky top-[var(--calendar-dates-sticky-offset,0px)] z-[80] flex-none border-b border-r sm:pr-8">
+      className={classNames(
+        "bg-default dark:bg-default sticky top-[var(--calendar-dates-sticky-offset,0px)] z-[80] flex-none sm:pr-8",
+        borderColor === "subtle" ? "border-b-subtle border-b" : "border-b-default border-b",
```

Comment:
Kinda seems like we should just allow classNames passed in for this, then we just handle it with over riding?